lldb/include/lldb/API/SBBlock.h

```diff
@@ -52,6 +53,8 @@ class LLDB_API SBBlock {
 
   lldb::SBAddress GetRangeEndAddress(uint32_t idx);
 
+  lldb::SBAddressRange GetRangeAtIndex(uint32_t idx);
+
```

Comment:
Use `SBAddressRangeList` here:
```
SBAddressRangeList GetRanges();
```

Reply:
Currently Block doesn't have GetRanges() function but it does have GetRangeAtIndex().
Would you like me to add GetRanges() to Block and then expose both, GetRanges() and GetRangeAtIndex(), to SBBlock